cmd/registry/patch/patch_test.go

```diff
@@ -25,7 +25,423 @@ func TestMain(m *testing.M) {
 	grpctest.TestMain(m, registry.Config{})
 }
 
+func TestApiPatches(t *testing.T) {
+	root := "projects/patch-api-test/locations/global"
+	tests := []struct {
+		resourceID string
+		parent     string
+		yamlFile   string
+		nested     bool
+		message    proto.Message
+	}{
+		{
+			resourceID: "registry",
```

Comment:
The tests are using the same resourceID, so the output of t.Run() won't be useful to identify which test failed. (Usually, we'd use a description of what is being tested.)

Reply:
I noticed that "#1" is appended for the second test, so left it at that - but if you'd prefer I'll make it explicit.

Reply:
ah. ok. thanks.